Resources/Locale/en-US/ghost/roles/ghost-role-component.ftl

```diff
@@ -19,3 +19,6 @@ ghost-role-information-hamlet-description = Lives in the station bridge, has a b
 
 ghost-role-information-slimes-name = Slime
 ghost-role-information-slimes-description = Everything around you irritates your instincts, destroy them!
+
+ghost-role-information-smile-name = Smile
+ghost-role-information-smile-description = The resident RND, a favorite specimen of experiments for scientists, because he survives. And he's so cute.
```

Comment:
What did you meant with "The resident RND"? see if you can word it a bit better like
```suggestion
ghost-role-information-smile-description = Science's best lab slime. Survived so many experiments that it got promoted to pet. 
```

Reply:
Correction. I'm sorry, I'm writing through a translator.